experimental/framework/fluid-static/src/containerCode.ts

```diff
@@ -142,11 +142,6 @@ export class RootDataObject
         throw new Error("Could not create new Fluid object because an unknown object was passed");
     }
 
-    public async getDataObject<T extends IFluidLoadable>(id: string) {
-        const handle = await this.root.wait<IFluidHandle<T>>(id);
```

Comment:
I believe the build is failing because `IFluidHandle` can now be removed as an import.